common/traits/00_traits.txt

```diff
@@ -2002,118 +2002,23 @@ cruel = {
 	opposites =  {
 		kind
 	}
-	
+
 	intrigue = 1
 	diplomacy = -1
 	combat_rating = 3 #new value!
-	
+
 	vassal_opinion = -5
 	ai_honor = -5
 	ai_greed = 5
-	
+
 	command_modifier = {
 		morale_offence = 0.1
 	}
-	
+
 	male_insult = INSULT_BRUTE
 	female_insult = INSULT_FIEND
 	child_insult = INSULT_TROLL
 	male_insult_adj = INSULT_CRUEL
 	female_insult_adj = INSULT_CALLOUS
 	child_insult_adj = INSULT_WICKED
 }
-
-#Leadership traits
-light_foot_leader = {
-	leader = yes
-	
-	command_modifier = {
-		light_infantry = 0.2
-	}
-}
-
-heavy_infantry_leader = {
-	leader = yes
-	
-	command_modifier = {
-		heavy_infantry = 0.2
-	}
-}
-
-cavalry_leader = {
```

Comment:
Remember:
You need to redo all references to light_foot_leader, heavy_infantry_leader and cavalry_leader in all game files that exist now or will be added in the future. Good luck...
...or you can do it much easier - delete infantry_commander_leader, use light_foot_leader instead, delete cavalry_officer_leader and use cavalry_leader instead, and do the same with heavy_infantry_leader.

Reply:
> You need to redo all references to light_foot_leader, heavy_infantry_leader and cavalry_leader in all game files that exist now or will be added in the future. Good luck...

Yep, and all spawned units. As an example, here's adventure army:
<details>
<summary>Click to expand</summary>
![image](https://user-images.githubusercontent.com/46384992/55170907-d740cf00-5190-11e9-8024-534893e61bc1.png)
</details>